### PR TITLE
Fix error level for win7 helix runs

### PR DESCRIPTION
**Pull request by HaoK**

### CODE REVIEW

File: .azure/pipelines/helix-matrix.yml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # We only want to run full helix matrix on main
-pr: none
+pr:
```
Comment (HaoK) [inline .azure/pipelines/helix-matrix.yml:2]: revert

**Comment by sebastienros** [comment]
The helix tests passed, is that the only things that is required?

**Comment by HaoK** [comment]
Yeah pretty much, if helix-matrix is green, that means things are good, they were failing before I fixed the errorlevel stuff

**Comment by HaoK** [comment]
They were failing like this before:

https://helixre8s23ayyeko0k025g8.blob.core.windows.net/dotnet-aspnetcore-refs-pull-32965-merge-b5ac0422a016476795/Diagnostics.FunctionalTests--net6.0/console.2b557351.log?sv=2019-07-07&se=2021-06-13T18%3A36%3A42Z&sr=c&sp=rl&sig=ZXAloi1RocpG34S7GP2LjNoxhM9SptBICUn%2FkW8ZiAw%3D

```
  Determining projects to restore...
neq was unexpected at this time.
"Finished running tests: exit_code=-2147483645"
```

**Comment by HaoK** [comment]
That said, I don't know if this is safe to merge since the Win7 issue was only on internal builds?  @dougbu can probably tell us whether adding the win7 queues back is something we want to skip for now.  But the change to the runtests.cmd is something that is good to fix now

**Comment by dougbu** [comment]
> That said, I don't know if this is safe to merge since the Win7 issue was only on internal builds? @dougbu can probably tell us whether adding the win7 queues back is something we want to skip for now. But the change to the runtests.cmd is something that is good to fix now

I'm confused because the restoration is in a block that's conditional on `'$(_UseHelixOpenQueues)' == 'true'` and I have not heard about any fix that would help here. If it's been fixed, wonderful. But, I don't want to see us needing to revert this change in two days.

IIRC the problem was common but not 💯% consistent. If that's the case, might make sense to run the aspnetcore-helix-matrix pipeline a couple more times. (/fyi Can revert the change to the YAML and just run the pipeline against your branch manually in AzDO. That allows some parallelization.)

**Comment by dougbu** [comment]
@HaoK please revert the .azure/pipelines/helix-matrix.yml changes and add information to the commit descriptions and the PR letting us know what your fixing and why. It's not clear as I read this how the fix is specific to Windows7.

**Comment by dougbu** [comment]
Minor but draft PRs are a great idea when the branch contains temporary changes such as those to .azure/pipelines/helix-matrix.yml in this PR.

**Comment by HaoK** [comment]
@sebastienros pointed out the runtests.cmd broke win7 in one his PRs, so this PR is specifically just to fix that, the only change I feel strongly about getting in is the runtests.cmd.  As that's what is causing the win7 issues.  Win7 was removed so I have to make those changes to verify the fix.

**Comment by HaoK** [comment]
It sounds like we aren't sure the win7 restore issues are resolved so I'll just back out those other changes then

**Comment by HaoK** [comment]
I don't know why this syntax fails only on win7, but the other windows queues have no problem with the original runtests.cmd

**Review by dougbu — APPROVED** [review]

**Comment by dougbu** [comment]
> It sounds like we aren't sure the win7 restore issues are resolved so I'll just back out those other changes then

Let's keep this separate for now. But, one approach might be to add a Win7 Helix test run to the quarantined PR for a while to confirm the `restore` status. A job that sets `$(HelixTargetQueues)` but otherwise matches our regular Helix x64 job (from the regular pipeline) might do it.

**Pull request merged**